config-templates/netapp/netapp-ontap-san/20.07/README.md

```diff
@@ -0,0 +1,96 @@
+   * Ensure the SVM has snapshot policies set up to be used by Trident. It is recommended a default snapshot policy is created on the SVM being used.
+
+## NetaApp Ontap-SAN Driver parameters & how to retrieve them
+
+Retrieve all parameters required by this template
+```
+ibmcloud sat storage template get --name netapp-ontap-san --version 20.7
+```
+
+**NetaApp Ontap-SAN Driver parameters**
+
+| Parameter | Required? | Description | Default value if not provided |
+| --- | --- | --- | --- |
+| `namespace` | Optional | Namespace for driver deployment | `trident-nas` |
+| `managementLIF` | Required | IP address of ONTAP management LIF | `N/A` |
+| `dataLIF` | Required | IP address of protocol LIF | `N/A` |
+| `svm` | Required | Storage virtual machine to use | `N/A` |
+| `username` | Required | Username to connect to the storage device | `N/A` |
+| `password` | Required | Password to connect to the storage device | `N/A` |
+| `limitVolumeSize` | Optional | Maximum requestable volume size and qtree parent volume size | `50Gi` |
+| `limitAggregateUsage` | Optional | Fail provisioning if usage is above this percentage | `80%` |
+
+## Default storage classes
+
+Provide a table of the Satellite storage classes that are installed when your configuration is assigned to a cluster. Refer to the following example table. Add or remove storage class details based on the storage type.
+
+| Storage class name | Type | File system | IOPs | Size range | Hard disk | Reclaim policy |
+| --- | --- | --- | --- | --- | --- | --- |
+| `sat-netapp-block-gold` | Ontap-SAN | Block | -- | -- | -- | Delete |
+| `sat-netapp-block-silver` | Ontap-SAN | Block | -- | -- | -- | Delete |
+| `sat-netapp-block-bronze` | Ontap-SAN | Block | -- | -- | -- | Delete | 
+
+
+## Creating the NetaApp Ontap-SAN Driver storage configuration
+
+Provide example steps, including an example `config create` command for creating a Satellite storage configuration that uses your template.
+
+**Example `sat storage config create` command**
+
+```
+ibmcloud sat storage config create --name ontapsan --template-name netapp-ontap-san --template-version 20.07 -p "managementLIF=10.0.0.1" -p "dataLIF=10.0.0.2" -p "svm=svm-nas" -p "username=admin" -p "password=admin-passW@rd"
+```
+
+## Creating the storage assignment
+
+**Example `sat storage assignment create` command**
+
+```
+ibmcloud sat storage assignment create --name sandriver --group blockstore --config ontapsan
+```
+
+## Verifying your NetaApp Ontap-NAS Driver storage configuration is assigned to your clusters
+
+Provide steps to retrieve any driver pods, storage classes, or any other resources that are deployed with your storage configuration.
+
+```
+oc get pods 
+```
+
+```
+oc get sc | grep sat
+```
+
+
+**Example output**
+
+Provide an example output screen of running driver pods and storage classes that are deployed by your configuration.
+
+
+## Troubleshooting
+
+Provide troubleshooting steps for any known issues.
+
```

Comment:
Are there any troubleshooting steps to add here? If we don't have troubleshooting yet, then remove this section.